README.md

```diff
@@ -337,6 +337,54 @@ IoT Core Shadows are human-readable JSON documents structured into `desired` and
 
 For the sample application, one node generates data, updating the `desired` section. The other node listens for updates to the `desired` section for its next target. Whenever it changes its state, it updates the `reported` state. The `delta` field is not used. The result is that the shadow shows an update to the `desired` digit every four seconds, with the `reported` digit moving towards the `desired` digit, alternating between upwards and downwards.
 
+### Sending ROS2 Logs to AWS CloudWatch
+
+The general strategy here is that we will send MQTT messages to a AWS IoT Rule using Basic Ingest, which in turn will forward the MQTT messages to Amazon Cloudwatch.
+
+We need to create an IAM role for the AWS IoT Rule. The first step is to set the permissions JSON file for the role.
+
+```
+export ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+
+export CWL_ROLE_PERMISSIONS_TEMPLATE="$(pwd)/templates/cwl_role_permissions_template.json"
+export CWL_ROLE_PERMISSIONS_FILE="$(pwd)/iot_certs_and_config/cwl_role_permissions.json"
```

Comment:
Please use the same path as the rest of the tutorial - `~/aws-iot-robot-connectivity-samples-ros2`

Reply:
fixed